### PR TITLE
Fix: Fixed to wait for multiple prompts

### PR DESCRIPTION
**Pull request by Yos-K**
Modified the prompt property of the Telnet structure to Vec<String> to listen for multiple prompts during the connection.
This is because the prompt changes when several commands are executed.
I would appreciate it if you could check the modified part of the code.

### CODE REVIEW

**Comment by kolapapa** [comment]
> Modified the prompt property of the Telnet structure to Vec to listen for multiple prompts during the connection. This is because the prompt changes when several commands are executed. I would appreciate it if you could check the modified part of the code.

First of all, thank you for your advice~

I think the API of the `prompts` method can be added directly. When `self.prompts` is set, `self.prompt` will be automatically overwritten, so that it can be compatible with the old version. What's your opinion?  (I think we prefer to pass `&str` when constructing `Telnet`, so we can directly convert it inside the API)

```rust
Telnet::builder().prompts(&["a", "b"])..
// is better than
Telnet::builder().prompts(vec!["a".to_string(), "b".to_string()])..
```

File: src/lib.rs [hunk 1]
```diff
@@ -25,8 +25,8 @@ pub struct TelnetBuilder {
 
 impl TelnetBuilder {
     /// Set the telnet server prompt, as many characters as possible.(`~` or `#` is not good. May misjudge).
-    pub fn prompt(mut self, prompt: &str) -> TelnetBuilder {
-        self.prompt = prompt.to_string();
+    pub fn prompts(mut self, prompts: Vec<String>) -> TelnetBuilder {
```
Comment (kolapapa) [inline src/lib.rs]:
```rust
pub fn prompts<T: ToString>(mut self, prompts: &[T]) -> TelnetBuilder
```
I think it's better here. What's your opinion?

**Comment by Yos-K** [comment]
I agree with you.
I did not take compatibility into consideration and I think your code is more preferable.

**Comment by kolapapa** [comment]
> I agree with you. I did not take compatibility into consideration and I think your code is more preferable.

Can you submit a new commit？:p

**Comment by Yos-K** [comment]
I have reflected your suggestion in the code and sent it. Please check it.
Also, in the doc comment for the login method, the port number was not specified as an argument, so I added it (it was a minor fix, so I included it in the same commit).

**Review by Yos-K — COMMENTED** [review]
Fixed to wait for multiple prompts

**Comment by kolapapa** [comment]
> Fixed to wait for multiple prompts

I think the `fn prompt()` can be keep it. Just add a new `fn prompts()`. Do you agree ？

**Comment by Yos-K** [comment]
> > Fixed to wait for multiple prompts
> 
> I think the `fn prompt()` can be keep it. Just add a new `fn prompts()`. Do you agree ？

Yes, either is fine with me as long as I can be waited on with multiple prompts.

**Comment by kolapapa** [comment]
@Yos-K Thanks~

**Pull request merged**